### PR TITLE
✨ Update `@percy/nightmare` to v2

### PR DESCRIPTION
**Pull request by Robdel12**
## What is this?

This updates the example app to the new CLI toolchain

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -1,100 +1,132 @@
...
+    new Promise((resolve) => {
+      xvfb.instance.stop(() => resolve());
+    })
+};
 
 describe('TodoMVC', function () {
-  this.timeout('10s')
+  let server, nightmare;
+
+  this.timeout('10s');
```
Comment (wwilsman) [inline test/test.js:26]: Is this timeout still necessary?

Comment (Robdel12) [reply test/test.js:26]: I think so, it's in their examples: https://github.com/segmentio/nightmare#examples

Comment (wwilsman) [reply test/test.js:26]: 🤷

**Review by wwilsman — APPROVED** [review]
🎮

**Pull request merged**